### PR TITLE
CMS-433: New project:update-info command and move org:analyze to this tool.

### PR DESCRIPTION
**Pull request by kporras07**
### Overview
This pull request adds new project:update-info command and moves org:analyze to this tool.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes 
| Has tests?    | no
| BC breaks?    | no     
| Deprecations? | no 

### Description
- Adds a mapping for Pantheon support level badges
- Adds new project:update-info command
- Move org:analyze command from hubph (and related utilities) into new OrgCommands file.

### CODE REVIEW

File: src/Cli/OrgCommands.php [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+use Hubph\Git\WorkingCopy;
+use Hubph\Git\Remote;
+
+class OrgCommands extends \Robo\Tasks implements ConfigAwareInterface, LoggerAwareInterface
+{
+    use ConfigAwareTrait;
+    use LoggerAwareTrait;
+
+    /**
+     * @command repo:convert-data
```
Comment (greg-1-anderson) [inline src/Cli/OrgCommands.php:28]: Are you still using this? It was part of my hackweek project, not sure we still need it.

File: src/Cli/ProjectCommands.php [hunk 2]
```diff
@@ -531,6 +531,153 @@ public function projectUpstreamUpdate($remote, $options = ['as' => 'default', 'p
...
+    /**
+     * Get right badge markdown.
+     */
+    protected function getSupportLevelBadge($level)
+    {
+        $badges = [
+            'ea' => '[![Early Access](https://img.shields.io/badge/pantheon-EARLY_ACCESS-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/early-access?q=org%3Apantheon-systems)',
+            'la' => '[![Limited Availability](https://img.shields.io/badge/pantheon-LIMITED_AVAILABILTY-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/limited-availability?q=org%3Apantheon-systems)',
+            'actively-supported' => '[![Actively Maintained](https://img.shields.io/badge/pantheon-actively_maintained-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/actively-maintained?q=org%3Apantheon-systems)',
+            'minimally-supported' => '[![Minimal Support](https://img.shields.io/badge/pantheon-minimal_support-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/minimal-support?q=org%3Apantheon-systems)',
```
Comment (greg-1-anderson) [inline src/Cli/ProjectCommands.php:673]: Maybe change the shortcuts to just `active` and `minimal` to keep the support column width down in the csv file.

**Review by greg-1-anderson — APPROVED** [review]
Looks good; had a couple minor comments.

**Comment by greg-1-anderson** [comment]
Looks like the test failure is just a simple code style check failure.

File: src/Cli/OrgCommands.php (hunk 1 above)
Comment (kporras07) [reply src/Cli/OrgCommands.php:28]: Nope. Deleting it

**Comment by kporras07** [comment]
All fixed. I'll merge when CircleCI passes

**Pull request merged**